kuksa_databroker/databroker/Cargo.toml

```diff
@@ -60,6 +55,9 @@ jemallocator = { version = "0.5.0", optional = true }
 lazy_static = "1.4.0"
 
 [features]
+default = ["tls"]
+tls = ["tonic/transport", "tonic/channel", "tonic/prost", "tonic/tls"]
+no-tls = ["tonic/transport", "tonic/channel", "tonic/prost"]
```

Comment:
Get rid of the `no-tls` feature, as features should never disable functionality.

Instead, rework the `tls` feature to add anything needed for TLS, i.e. enabling TLS.

See:
https://doc.rust-lang.org/cargo/reference/features.html#feature-unification

> [...] features should be additive. That is, enabling a feature should not disable functionality, and it should usually be safe to enable any combination of features. [...]

> For example, if you want to optionally support [no_std](https://doc.rust-lang.org/reference/names/preludes.html#the-no_std-attribute) environments, do not use a no_std feature. Instead, use a std feature that enables std.

There's even a clippy warning for this:
https://github.com/rust-lang/rust-clippy/pull/7539

Reply:
@argerus Thanks you for the comment, the changes are done.